qa/tasks/ceph_test_case.py

```diff
@@ -108,7 +108,8 @@ def assert_cluster_log(self, expected_pattern, invert_match=False,
 
         class ContextManager(object):
             def match(self):
-                found = expected_pattern in self.watcher_process.stdout.getvalue()
+                line = self.watcher_process.stdout.getvalue()
```

Comment:
`line` is not used anywhere else, so, why break this up into two?

Reply:
Was useful for an earlier patch. I've removed it.